### DISCUSSION TITLE
Is there any guide or way to accept request body as xml?

### DISCUSSION THREAD

**Discussion by rohts-patil** [fastapi/fastapi#8773] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
NA
```

### Description

Whenever I am designing post api, I need my body in xml form. Is there any way where the xml get converted my model class?API

### Operating System

macOS

### Operating System Details

macOS Catalina 10.15.7

### FastAPI Version

0.68.0

### Python Version

3.8

**Comment by STeveShary** [comment]
I don't know if there is a "simple" way to do it.  I think that if you bring in the body as a string, convert to a dict and then you can create the model.  Here is a xml -> dict library I have used: https://pypi.org/project/xmltodict/

These are the steps to look at:
- Get the request body directly as a string: https://fastapi.tiangolo.com/advanced/using-request-directly/?h=request+body
- Convert the XML string to a dict: https://pypi.org/project/xmltodict/
- Build a Pydantic Model object using named parameters.

It's not perfect, but it would work.

**Comment by rohts-patil** [comment]
Yes, that is what I have done as of now. Thanks
